lib/exabgp/bgp/message/update/attribute/mprnlri.py

```diff
@@ -171,6 +174,10 @@ def unpack (cls, data, negotiated):
 				length,nlri = NLRI.unpack(afi,safi,data,addpath,'',IN.ANNOUNCED)
 				nlris.append(nlri)
 
+			if length == 0:
```

Comment:
I the the subclass on which we called unpack has, for some reason, returned a zero length, then unless we do soemthing like the above, we will call it again with the same data and it will return zero again... loop... :(

So let's consider instead that if len==0 then it means it has consumed everything...